src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlServerTransparentDataEncryptionCertificate.md

```diff
@@ -0,0 +1,203 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/Add-AzureRmSqlServerTransparentDataEncryptionCertificate
+schema: 2.0.0
+---
+
+# Add-AzureRmSqlServerTransparentDataEncryptionCertificate
+
+## SYNOPSIS
+Adds a Transparent Data Encryption Certificate for the given SQL Server instance
+
+## SYNTAX
+
+### AddAzureRmSqlServerTransparentDataEncryptionCertificate Input Object Parameter Set
+```
+Add-AzureRmSqlServerTransparentDataEncryptionCertificate [-SqlServer] <AzureSqlServerModel>
+ [-PrivateBlob] <String> [-Password <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### AddAzureRmSqlServerTransparentDataEncryptionCertificate Resource Id Parameter Set
+```
+Add-AzureRmSqlServerTransparentDataEncryptionCertificate [-SqlServerResourceId] <String> [-PrivateBlob] <String>
+ [-Password <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### AddAzureRmSqlServerTransparentDataEncryptionCertificate Default Parameter Set
+```
+Add-AzureRmSqlServerTransparentDataEncryptionCertificate [-ResourceGroupName] <String> [-ServerName] <String>
+ [-PrivateBlob] <String> [-Password <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Add-AzureRmSqlManagedInstanceTransparentDataEncryptionCertificate adds a Transparent Data Encryption Certificate for the given SQL Server instance
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\>     Add-AzureRmSqlServerTransparentDataEncryptionCertificate -ResourceGroupName "YourResourceGroupName" -ServerName "YourServerName" -PrivateBlob "MIIJ+QIBAzCCCbUGCSqGSIb3DQEHAaCCCaYEggmiMIIJnjCCBhcGCSqGSIb3Dasdsadasd" -Password "CertificatePassword"
+PS C:\>     $server | Add-AzureRmSqlServerTransparentDataEncryptionCertificate -PrivateBlob "MIIJ+QIBAzCCCbUGCSqGSIb3DQEHAaCCCaYEggmiMIIJnjCCBhcGCSqGSIb3Dasdsadasd"
+PS C:\>     Add-AzureRmSqlServerTransparentDataEncryptionCertificate -SqlServerResourceId $server.ResourceId -PrivateBlob "MIIJ+QIBAzCCCbUGCSqGSIb3DQEHAaCCCaYEggmiMIIJnjCCBhcGCSqGSIb3Dasdsadasd"
+```
+
```

Comment:
please split into 3 separate examples with comments, for example: 
### Example 2
```Powershell
Get-AzureRmSqlServer | Add-AzureRmSqlServerTransparentDataEncryptionCertificate -ResourceGroupName <ResourceGroupName> -PrivateBlob <PrivateBlob>  -Password <Password>
```
Add TDE certificate to all sql servers in a resource group

### Example 3
```Powershell
$server = Get-AzureRmSqlServer -ServerName <ServerName> -ResourceGroupName <ResourceGroupName> 
Add-AzureRmSqlServerTransparentDataEncryptionCertificate -SqlServerResourceId $server.ResourceId -PrivateBlob "MIIJ+QIBAzCCCbUGCSqGSIb3DQEHAaCCCaYEggmiMIIJnjCCBhcGCSqGSIb3Dasdsadasd"
<Password>
```
Add TDE certificate to the servers using server resourceId